stubs/Builder.stubphp

```diff
@@ -210,6 +209,11 @@ class Builder
      */
     public function has($relation, string $operator = '>=', int $count = 1, string $boolean = 'and', Closure $callback = null) {}
 
+    /**
+     * @return int
+     */
```

Comment:
Move to the php signature please. This is stub file so there is no need to maintain interface limitations, etc.